stat.c

```diff
@@ -3087,17 +3091,29 @@ int calc_log_samples(void)
 		if (!td->bw_log ||
 			(td->bw_log && !per_unit_log(td->bw_log))) {
 			tmp = add_bw_samples(td, &now);
-			if (tmp < next)
-				next = tmp;
+
+			if (td->bw_log)
+				log_avg_msec = td->bw_log->avg_msec;
 		}
 		if (!td->iops_log ||
 			(td->iops_log && !per_unit_log(td->iops_log))) {
 			tmp = add_iops_samples(td, &now);
-			if (tmp < next)
-				next = tmp;
+
+			if (td->iops_log)
+				log_avg_msec = td->iops_log->avg_msec;
 		}
+
+		if (tmp < next)
+			next = tmp;
 	}
 
+	if (log_avg_msec == 0)
+		next_mod = elapsed_time;
```

Comment:
So this say the time that started since the job started... Won't get bigger steadily?

Reply:
Yes, next_mod will get steadily bigger here, but the next calculation on line 3115 is still correct.

Reply:
Ah I see!